manifests/init.pp

```diff
@@ -71,111 +71,115 @@
+  $zabbix_template_dir                                                    = $zabbix::params::zabbix_template_dir,
+  $zabbix_web_ip                                                          = $zabbix::params::zabbix_web_ip,
+  Zabbix::Databases $database_type                                        = $zabbix::params::database_type,
+  $database_path                                                          = $zabbix::params::database_path,
+  $manage_database                                                        = $zabbix::params::manage_database,
+  $default_vhost                                                          = $zabbix::params::default_vhost,
+  $manage_vhost                                                           = $zabbix::params::manage_vhost,
+  $manage_firewall                                                        = $zabbix::params::manage_firewall,
+  $manage_repo                                                            = $zabbix::params::manage_repo,
+  $manage_resources                                                       = $zabbix::params::manage_resources,
+  $manage_service                                                         = $zabbix::params::manage_service,
+  $apache_use_ssl                                                         = $zabbix::params::apache_use_ssl,
+  $apache_ssl_cert                                                        = $zabbix::params::apache_ssl_cert,
+  $apache_ssl_key                                                         = $zabbix::params::apache_ssl_key,
+  $apache_ssl_cipher                                                      = $zabbix::params::apache_ssl_cipher,
+  $apache_ssl_chain                                                       = $zabbix::params::apache_ssl_chain,
+  $apache_listen_ip                                                       = $zabbix::params::apache_listen_ip,
+  $apache_listenport                                                      = $zabbix::params::apache_listenport,
+  $apache_listenport_ssl                                                  = $zabbix::params::apache_listenport_ssl,
+  $apache_php_max_execution_time                                          = $zabbix::params::apache_php_max_execution_time,
+  $apache_php_memory_limit                                                = $zabbix::params::apache_php_memory_limit,
+  $apache_php_post_max_size                                               = $zabbix::params::apache_php_post_max_size,
+  $apache_php_upload_max_filesize                                         = $zabbix::params::apache_php_upload_max_filesize,
+  $apache_php_max_input_time                                              = $zabbix::params::apache_php_max_input_time,
+  $apache_php_always_populate_raw_post_data                               = $zabbix::params::apache_php_always_populate_raw_post_data,
+  Optional[Stdlib::Absolutepath] $ldap_cacert                             = $zabbix::params::ldap_cacert,
```

Comment:
can you add a test that sets those parameters and ensure that the file is rendered correctly?